src/app/App.tsx

```diff
@@ -1,20 +1,20 @@
 import React, { Suspense } from "react";
 import * as ReactDOM from "react-dom";
 import { StylesProvider } from "@material-ui/core/styles";
-import { CheckServerConnectionComponent } from "./Main/CheckServerConnection.component";
```

Comment:
I think not that checking the server connection in general is useless. As discussed there should be a mechanism connected to the loading of necessary data from the server on page load, that shows a spinner and/or displays a message when the loading failed. That is because of loading the web page and accessing the api are different mechanism or parts of the server and if the api part fails the user should be informed as soon as possible about that. So i think you already have that mechanism.

Reply:
But for this situation, I have the error page, right?

Reply:
If it is triggered if the page is loaded and not only after the user navigates through the app, yes.